### DISCUSSION TITLE
Cannot log in

### DISCUSSION THREAD

**Discussion by zenithbrew** [open-webui/open-webui#2207] [General]
# Bug Report

## All emails fails

**Bug Summary:**
Account Activation Pending
Contact Admin for WebUI Access
Your account status is currently pending activation. To access the WebUI, please reach out to the administrator. Admins can manage user statuses from the Admin Panel.

**Steps to Reproduce:**
Load docker w/ ollama up in ubuntu

**Expected Behavior:**
should log in

**Actual Behavior:**
Stuck @

"Account Activation Pending
Contact Admin for WebUI Access
Your account status is currently pending activation. To access the WebUI, please reach out to the administrator. Admins can manage user statuses from the Admin Panel.
"

## Environment
Ubuntu latest

Browser:firefox latest

![Screenshot from 2024-05-11 15-37-36](https://github.com/open-webui/open-webui/assets/156859584/6a49cff2-fd65-425a-b44e-6afd32b9b2c1)

**Comment by silentoplayz** [comment]
https://docs.openwebui.com/troubleshooting/

**Comment by pasmud** [comment]
Go to app/backend/data folder, delete webui.db and restart the app. If in docker do the same and restart the container. ** This will create a new DB, so start with a new admin, account. Other option would be to reset the admin pass according to doc.

**Reply by medledan** [reply]
3 years later I find this random comment. This worked. This solved my issue. Thanks buddy.
